res/res_pjsip_endpoint_identifier_ip.c

```diff
@@ -241,6 +264,42 @@ static int header_identify_match_check(void *obj, void *arg, int flags)
 	return 0;
 }
 
+/*! \brief Comparator function for matching an object by request line */
+static int request_identify_match_check(void *obj, void *arg, int flags)
+{
+	struct ip_identify_match *identify = obj;
+	struct pjsip_rx_data *rdata = arg;
+
+	if (ast_strlen_zero(identify->match_request)) {
+		return 0;
+	}
+
+	int len;
```

Comment:
This won't build under dev mode, these need to be at the start of the function.

Reply:
Moved the local variable definitions before the actual code. I think that was the problem.